### ISSUE TITLE
Axe is reporting multiple <label>s warning with aria-hidden=true

### ISSUE THREAD

**Issue by starzonmyarmz** [dequelabs/axe-core#2103]
Axe (Firefox extension) gives me the following warning: _Multiple label elements is not widely supported in assistive technologies. Ensure the first label contains all necessary information._ with the following code:

```
<input type="text" id="test">

<label for="test">Label one</label>, 
<label for="test" aria-hidden="true">Label two</label>, 
<label for="test" aria-hidden="true">Label three</label>
```

Expectation: Axe would not throw this warning because it would ignore the `<label aria-hidden="true">`s

Result: Axe throws a warning.

JS Fiddle: https://jsfiddle.net/qjy54fz8/2/

**Comment by straker** [comment]
The warning is the expected result in this case. We [tested various patterns of multiple labels](https://codepen.io/straker/pen/PvqONy) and Talkback on Android did not respect the `aria-hidden`. So we give a warning to let you know.
